src/request.ts

```diff
@@ -55,6 +55,8 @@ export interface ISubmitRequestLinkFulfillmentProps {
 	signedTx?: string
 	apiUrl?: string
 	link: string
+	estimatedPoints: string
```

Comment:
why is estimatedPoints necessary? and why is it Obligatory?
Do we handle this the same way with claimLinks? Please check how we do it there